v2/api/microsoft.authorization/v1alpha1api20200801preview/role_assignment_types_gen.go

```diff
@@ -261,6 +270,9 @@ func (roleAssignment *RoleAssignment) AssignPropertiesToRoleAssignment(destinati
 	}
 	destination.Status = status
 
+	// TypeMeta
+	destination.TypeMeta = roleAssignment.TypeMeta
```

Comment:
`TypeMeta` contains the `Kind` _and_ the `APIVersion`. Is it safe to copy this to a type with a different version? Obviously the `Kind` copy is safe, but the `APIVersion` copy seems suspect since the `APIVersion` of the destination is not the same as the source?

Reply:
Indeed! I had changes to only copy `ObjectMeta` that I hadn't pushed. I've rebased and updated.